cpu/esp_common/esp-wifi/esp_wifi_netdev.c

```diff
@@ -903,6 +903,62 @@ static wifi_config_t wifi_config_ap = {
 };
 #endif /* (defined(MCU_ESP8266) && !defined(MODULE_ESP_NOW)) || defined(MODULE_ESP_WIFI_AP) */
 
+
+#if (defined(MCU_ESP32) && defined(MODULE_ESP_WIFI_CSI))
+void __attribute__((weak)) esp_wifi_csi_rx_cb(void *ctx, wifi_csi_info_t *info)
+{
+    if (!info || !info->buf) {
+        ESP_WIFI_LOG_ERROR("ERROR: esp_wifi_csi_rx_cb missing info or buf");
+        return;
+    }
+    printf("{");
+    printf("\"mac\": \"%02x:%02x:%02x:%02x:%02x:%02x\",",
+           (unsigned)info->mac[0], (unsigned)info->mac[1], (unsigned)info->mac[2],
+           (unsigned)info->mac[3], (unsigned)info->mac[4], (unsigned)info->mac[5]);
```

Comment:
```suggestion
    printf("\"mac\": \"" MAC_STR "\",", MAC_STR_ARG(mac));
```

Reply:
Somehow I couldn't the header containing the macro but I will give it another look.

Reply:
These macros are defined in the file itself here https://github.com/RIOT-OS/RIOT/blob/be77d87480c12acbbfedc67f1ca9c56a08fa2151/cpu/esp_common/esp-wifi/esp_wifi_netdev.c#L71-L72

Reply:
I defined them because they are used at several places in this file.

Reply:
I feel like I asked someone where my glasses are when they were on my head the whole time :laughing: